### PR TITLE
I think it's better to not imply false statements

### PR DESCRIPTION
**Pull request by radamar**
My arguments on webmaster@python.org:
A program operates in its own Universe?
- First, the program is operating on a computer, which is based on physical laws, same ones that require enormous masses to be round.

- Second, It's not adding any knowledge. Purpose of a lesson is to impart knowledge, implying obviously false statements is contradictory to purpose of a lesson.

- Third, I think the person who contributed the lesson was just trying to be edgy or sarcastic, which should be corrected out. A tutorial should be properly vetted. At the expense of making the joke, you(not you, but you are not helping resolve it) are lengthening the lesson.

So, I insist.

### CODE REVIEW

**Comment by z3ntu** [comment]
Just so btw I looked through the python source and found that that text was merged in cvs revision number `1.104` on the 3rd of April 2000 into the release152p1-patches branch(?)
```
revision 1.104
date: 2000/04/03 04:26:58;  author: fdrake;  state: Exp;  lines: +410 -267

Merge in changes from the 1.5.2p2 release.
```
Unfortunately I couldn't find the revision of the original change (cvs is not really user friendly and I've never used it before...) but close enough, I guess

**Comment by serhiy-storchaka** [comment]
1. You have to create a pull request to the master branch.
2. `the_world_is_flat == False` is an example of extremely bad style.
3. Python is full of jokes.

File: Doc/tutorial/interpreter.rst [hunk 1]
```diff
@@ -109,11 +109,11 @@ before printing the first prompt:
 Continuation lines are needed when entering a multi-line construct. As an
 example, take a look at this :keyword:`if` statement::
 
-   >>> the_world_is_flat = True
-   >>> if the_world_is_flat:
-   ...     print("Be careful not to fall off!")
+   >>> the_world_is_flat = False
+   >>> if the_world_is_flat == False:
```
Comment (datalogics-robb) [inline Doc/tutorial/interpreter.rst:113]: I hope this review serves to educate.  While the change seems innocuous, it includes one example of a bad idiom that I have seen repeated many times.

Testing equality against a global singleton is bad, testing equality against boolean literal is worse.  The statement:

     if the_world_is_flat:

implies nothing about the type stored in this variable, and is NOT equivalent to the statement:

     if the_world_is_flat == True:

There is the concept of "truthieness" of any value, 0, None, 0.0, and False will not evaluate to "truth", conversely nonzero numbers will.

Comment (radamar) [reply Doc/tutorial/interpreter.rst:113]: Yes, thanks. I am learning afterall.

**Pull request closed without merging**